// @aws-sdk/client-braket/dist-cjs/Braket.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Braket = void 0;
const smithy_client_1 = require("@smithy/smithy-client");
const BraketClient_1 = require("./BraketClient");
const CancelJobCommand_1 = require("./commands/CancelJobCommand");
const CancelQuantumTaskCommand_1 = require("./commands/CancelQuantumTaskCommand");
const CreateJobCommand_1 = require("./commands/CreateJobCommand");
const CreateQuantumTaskCommand_1 = require("./commands/CreateQuantumTaskCommand");
const GetDeviceCommand_1 = require("./commands/GetDeviceCommand");
const GetJobCommand_1 = require("./commands/GetJobCommand");
const GetQuantumTaskCommand_1 = require("./commands/GetQuantumTaskCommand");
const ListTagsForResourceCommand_1 = require("./commands/ListTagsForResourceCommand");
const SearchDevicesCommand_1 = require("./commands/SearchDevicesCommand");
const SearchJobsCommand_1 = require("./commands/SearchJobsCommand");
const SearchQuantumTasksCommand_1 = require("./commands/SearchQuantumTasksCommand");
const TagResourceCommand_1 = require("./commands/TagResourceCommand");
const UntagResourceCommand_1 = require("./commands/UntagResourceCommand");
const commands = {
    CancelJobCommand: CancelJobCommand_1.CancelJobCommand,
    CancelQuantumTaskCommand: CancelQuantumTaskCommand_1.CancelQuantumTaskCommand,
    CreateJobCommand: CreateJobCommand_1.CreateJobCommand,
    CreateQuantumTaskCommand: CreateQuantumTaskCommand_1.CreateQuantumTaskCommand,
    GetDeviceCommand: GetDeviceCommand_1.GetDeviceCommand,
    GetJobCommand: GetJobCommand_1.GetJobCommand,
    GetQuantumTaskCommand: GetQuantumTaskCommand_1.GetQuantumTaskCommand,
    ListTagsForResourceCommand: ListTagsForResourceCommand_1.ListTagsForResourceCommand,
    SearchDevicesCommand: SearchDevicesCommand_1.SearchDevicesCommand,
    SearchJobsCommand: SearchJobsCommand_1.SearchJobsCommand,
    SearchQuantumTasksCommand: SearchQuantumTasksCommand_1.SearchQuantumTasksCommand,
    TagResourceCommand: TagResourceCommand_1.TagResourceCommand,
    UntagResourceCommand: UntagResourceCommand_1.UntagResourceCommand,
};
class Braket extends BraketClient_1.BraketClient {
}
exports.Braket = Braket;
(0, smithy_client_1.createAggregatedClient)(commands, Braket);
